app/models/flavor.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class Flavor < ApplicationRecord
+  STOCK_STATUSES = {
+    no: 'no',
+    yes: 'yes'
+  }.freeze
+
+  enum :instock, STOCK_STATUSES, default: STOCK_STATUSES[:yes]
```

Comment:
@gullariz12 what do you think about this? Although I don't think we will use this. Most probably we will have `instock` method which will use `quantity` to decide if it's in stock or not.
```suggestion
  enum :instock, [:yes, :no], default: :yes
```

Reply:
Agreed with your thoughts. I will remove it from the DB as well we will calculate it on runtime.
Actually, I reference it from the previous scheme and think that we will add a field for it.